### PR TITLE
Update Markdown of OSS.

### PR DESCRIPTION
**Pull request by Booster1212**

### CODE REVIEW

File: plugins/derlord/OSS.md [hunk 1]
```diff
@@ -36,71 +41,66 @@ So feel free to do whatever you want!
...
-    { sprite: 52, color: 1, text: 'Seller Example - Index 0', scale: 1 }, // Index 0
+export const coreShopLocations: IShopLocation[] = [
+    { x: 25.980966567993164, y: -1345.6417236328125, z: 28.497024536132812, isBlip: true } as IShopLocation, 
+    { x: 374.3475341796875, y: 328.112060546875, z: 102.56637573242188, isBlip: true } as IShopLocation, 
+    { x: -3041.32763671875, y: 585.155029296875, z: 6.908928871154785, isBlip: true } as IShopLocation, 
+    { x: -3243.743408203125, y: 1001.3903198242188, z: 11.830706596374512, isBlip: true } as IShopLocation, 
+    { x: 548.0447387695312, y: 2669.48876953125, z: 41.156490325927734, isBlip: true } as IShopLocation, 
+    { x: 1960.2322998046875, y: 3742.317138671875, z: 31.343746185302734, isBlip: true } as IShopLocation, 
+    { x: 1730.01171875, y: 6416.22021484375, z: 34.03722381591797, isBlip: true } as IShopLocation, 
+    { x: 2555.4609375, y: 382.1643371582031, z: 107.62295532226562, isBlip: true } as IShopLocation,
```
Comment (leonardssh) [inline plugins/derlord/OSS.md]: Is there a specific purpose why you used as `IShopLocation` for each item? Once you declare the type of the variable in question, its children will take over the type itself.

I don't see the point in overtyping.

File: plugins/derlord/OSS.md [hunk 2]
```diff
@@ -36,71 +41,66 @@ So feel free to do whatever you want!
...
-// Shops are based on the index of arrays so -> First index -> First sellList
-export const SELLERS: alt.Vector3[] = [
-    { x: 1163.400634765625, y: -323.938232421875, z: 68.20509338378906 } as alt.Vector3 // SELLER (SHOP-POS) INDEX -> 0
-    // { Add stuff yourself } Just another seller, INDEX 1 -> The secoond list will get into this position.
-];
-
-// Used to give all the shops are different blip, just copy the line as often as you need it.
-export const sellersBlips = [
-    { sprite: 52, color: 1, text: 'Seller Example - Index 0', scale: 1 }, // Index 0
+export const coreShopLocations: IShopLocation[] = [
```
Comment (leonardssh) [inline plugins/derlord/OSS.md:58]: Shouldn't we also define the types themselves?

File: plugins/derlord/OSS.md [hunk 3]
```diff
@@ -36,71 +41,66 @@ So feel free to do whatever you want!
...
+    { x: 2555.4609375, y: 382.1643371582031, z: 107.62295532226562, isBlip: true } as IShopLocation, 
 ];
 
-const burgerBuyer = [
-    { name: 'Burger', dbName: 'burger', price: 330, image: 'crate' }, // SellList INDEX -> 0
-    // { Add stuff yourself. } Second Items for SELLERS Index -> 0
-];
-const anotherExampleList = [
-    // { Fill me! } -> Index 1
+export const ShopRegistry: Array<IShop> = [
```
Comment (leonardssh) [inline plugins/derlord/OSS.md]: Same here, where's the type `IShop` defined?

**Review by leonardssh — CHANGES_REQUESTED** [review]

File: plugins/derlord/OSS.md (hunk 1 above)
Comment (leonardssh) [reply plugins/derlord/OSS.md]: See in [playground](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgJIGUAWB7ADgGWwTjGGxGQG8AoZO5ADwC5kQBXAWwCNoBuW+gE8W7bnwF0AXiM48o-esmABnAEIAbYLhZds2dRDgh+AX2rUE5ZWGSWoELHkLFSVlhhwEiJMiADaALrIALzIfooSVIwsAEwArAB0AJwAHAAMSQBsmXGZAOxJSQDMAIyZACwANMjCyAC0JUXliRUleTFFmUUxKSXx1dLIPQnlSXlpMc2djT191SoaWixgUGwoJtWRlNHIRXnlCU15cU1tWSnH1bXdKQklfWmZaXHlmRdxAywlEwm5Xcd5brlGIlFIpeZqTTaZArNbIMwBfgWKz6CAJdTYADmAAo7A5PM4fFY-GkAglBABKXhAA)

File: plugins/derlord/OSS.md (hunk 3 above)
Comment (Booster1212) [reply plugins/derlord/OSS.md]: Will be resolved soon. Thanks 👍

File: plugins/derlord/OSS.md (hunk 1 above)
Comment (Booster1212) [reply plugins/derlord/OSS.md]: Overtypes is resolved soon as well, thanks. 👍

File: plugins/derlord/OSS.md (hunk 2 above)
Comment (Booster1212) [reply plugins/derlord/OSS.md:58]: Unsure what this exactly means? 🤔

**Review by leonardssh — APPROVED** [review]

File: plugins/derlord/OSS.md [hunk 4]
```diff
@@ -36,71 +41,67 @@ So feel free to do whatever you want!
...
+        locations: sellerExampleLocations,
+    },
 ];
-
-export const sellLists = [burgerBuyer, /*anotherExampleList*/]; // Dont forget to add custom item lists here
-
 ```
 
-<img src="https://user-images.githubusercontent.com/82890183/148910952-470985fe-5fed-41ed-8b87-08c9977f71c2.png" width="500"/>
+<img src="https://user-images.githubusercontent.com/82890183/148910952-470985fe-5fed-41ed-8b87-08c9977f71c2.png" width="6500"/>
```
Comment (Stuyk) [inline plugins/derlord/OSS.md]: Why is this 6500?

**Review by Stuyk — CHANGES_REQUESTED** [review]

File: plugins/derlord/OSS.md (hunk 4 above)
Comment (Booster1212) [reply plugins/derlord/OSS.md]: Fixd

**Review by Stuyk — APPROVED** [review]

**Pull request merged**